docs/sources/configure-client/grafana-alloy/receive_profiles.md

```diff
@@ -0,0 +1,194 @@
+---
+title: Receive profiles from Pyroscope SDKs
+menuTitle: Receive SDK profiles
+description: Learn how to configure Grafana Alloy to receive profiles from applications using Pyroscope SDKs.
+weight: 10
+---
+
+# Receive profiles from Pyroscope SDKs
+
+The `pyroscope.receive_http` component in Alloy receives profiles from applications instrumented with Pyroscope SDKs. This approach provides several benefits:
+- Lower latency by sending profiles to a local Alloy instance instead of over internet
+- Separation of infrastructure concerns (auth, routing) from application code
+- Centralized management of authentication and metadata enrichment
+
+To set up profile receiving, you need to:
+1. Configure Alloy components
+2. Configure your application's SDK
+3. Start Alloy
+
+## Configure Alloy components
+
+The configuration requires at least two components:
+- `pyroscope.receive_http` to receive profiles via HTTP
+- `pyroscope.write` to forward profiles to Pyroscope
+
+Here's a basic configuration:
```

Comment:
Should we explain what this basic configuration provides?

Reply:
Updated in 3482321765aeb1bb64e627bf28fe48ea0b72cbb2.